bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-notes/styles.scss

```diff
@@ -73,17 +73,20 @@
   box-shadow: inset 0 0 0 var(--border-size) var(--item-focus-border), inset 1px 0 0 1px var(--item-focus-border);
 }
 
-.notification {
-  position: relative;
+.unreadContent {
+  @extend %flex-column;
+  justify-content: center;
+}
 
-  &:after {
-    content: '';
-    position: absolute;
-    border-radius: 50%;
-    width: 8px;
-    height: 8px;
-    bottom: 3px;
-    right: 3px;
-    background-color: var(--color-danger);
-  }
+.unreadContentText {
```

Comment:
Note to other reviewers - this is largely shared with chat-unread-messages' .unreadMessagesText (hence the numbers)